experiments/AMIP/modular/components/ocean/slab_ocean_init.jl

```diff
@@ -72,3 +86,15 @@ end
 Ensures that the space of the SST struct matches that of the land_fraction, and converts the units to Kelvin (N.B.: this is dataset specific)
 """
 clean_sst(SST, _info) = (swap_space!(zeros(axes(_info.land_fraction)), SST) .+ float_type_bcf(_info)(273.15))
+
+# required by Interfacer
+get_field(sim::SlabOceanSimulation, ::Val{:air_temperature}) = sim.integrator.u.T_sfc
+get_field(sim::SlabOceanSimulation, ::Val{:z0m}) = sim.integrator.p.params.z0m
+get_field(sim::SlabOceanSimulation, ::Val{:z0b}) = sim.integrator.p.params.z0b
+get_field(sim::SlabOceanSimulation, ::Val{:beta}) = convert(eltype(sim.integrator.u), 1.0)
+get_field(sim::SlabOceanSimulation, ::Val{:albedo}) = sim.integrator.p.params.α
+get_field(sim::SlabOceanSimulation, ::Val{:area_fraction}) = sim.integrator.p.area_fraction
```

Comment:
The `SlabOceanSimulation` and `PrescribedIceSimulation` models should be perceived as individual models, which have their own independent data structures. I think this way it will be less confusing when swapping them for other models / modifying them. But if we decide to support these models as a long-term stable solution, then we should probably create a new module for them. I'd be in favor of deciding this as part of ClimaEarth though...